docs/install/install-script.md

```diff
@@ -7,6 +7,8 @@
 
 - **Kubectl and Kustomize** - The installation will occur via the terminal using [kubectl](https://kubernetes.io/docs/tasks/tools/#kubectl) and [kustomize](https://kubectl.docs.kubernetes.io/installation/kustomize/).
 
+- **jq** - The `--enable-self-signed-ca` flag depends on the installation of [jq](https://jqlang.github.io/jq/download/)
```

Comment:
I'd say it's not needed here, these are more general pre-reqs for a from-scratch install/deployment.